test/deps.sh

```diff
@@ -17,6 +17,7 @@ name () {
 test () {
   echo "## TESTING DEPENDENCY: $1"
   pushd node_modules/$1
+  npm install --only=dev
   npm test | name $1
```

Comment:
I wasn't use how to merge this line.
Is currently:

```bash
npm test | name $1
```

On this branch is was originally: 
```bash
npm test
```